src/Parse/ParseQuery.php

```diff
@@ -623,6 +623,26 @@ public function withinGeoBox($key, $southwest, $northeast)
     }
 
     /**
+     * Add a constraint to the query that requires a particular key's
+     * coordinates be contained within a given polygon
+     *
+     * @param string        $key       The key of the ParseGeoPoint
+     * @param array         $points    Array of ParseGeoPoint
+     *
+     * @return ParseQuery Returns this query, so you can chain this call.
+     */
+    public function withinPolygon($key, $points)
+    {
+        $this->addCondition(
+            $key,
+            '$geoWithin',
+            ['$polygon' => $points]
+        );
+
+        return $this;
+    }
+
```

Comment:
@dplewis I did not. I run all tests against mongodb by default. I'm making an assumption that database integrations are sufficiently properly vetted/tested on the server side of things to ensure that using an alternate db would not change behavior in the API. 

However that's not always the case in reality. Did you find something running against an alternate db or were you just checking?